efb_mp_instantview_middleware/telegraph.py

```diff
@@ -4,6 +4,17 @@
 from bs4 import BeautifulSoup as bs
 from bs4 import Tag, NavigableString
 
+def stripKey(dict, key):
+        if type(dict).__name__=="dict":
+                for ownkey in dict.copy():
+                        if ownkey==key:
+                                dict.__delitem__(key)
+                        else:
+                                stripKey(dict[ownkey], key)
+        elif type(dict).__name__=='list':
+                for n in range(len(dict)):
+                        stripKey(dict[n],key)
```

Comment:
```suggestion
def stripKey(node, key):
    if isinstance(node, dict):
        for ownkey in tuple(node.keys()):
            if ownkey == key:
                del node[key]
            else:
                stripKey(node[ownkey], key)
    elif isinstance(node, list):
        for sub_node in node:
            stripKey(sub_node, key)
```

Are you a js developer?

Disclaimer: I cannot guarantee that my version should work. I am just telling you how to write pythonic code.

Reply:
Thanks for your suggestion, just learned.
I am just interested in it and kinds of Obsessive-compulsive disorder.